tests/integration/README.md

```diff
@@ -38,10 +38,14 @@ deleted manually.
 Copy `.env.sample` to `.env`, put the respective provider configurations in the
 environment variables and source it, `source .env`.
 
-Run the test with `make test-*`:
+Ensure the image-reflector-controller container image to be tested is built and
+ready for testing.
+
```

Comment:
Should we add a small note here on how to build the `image-controller-image`?

Reply:
I think a brief pointer to the Makefile at the root of the repo would be good.

I was also thinking of adding a check if the given `TEST_IMG` exists in the local docker image store and if not, fail. If we do that, I think then we can put a default value for `TEST_IMG`, and print a message that it doesn't exist when it's not explicitly set.

Sure, we can also add some instructions about testing a stable release image.

Reply:
> I was also thinking of adding a check if the given TEST_IMG exists in the local docker image store and if not, fail.

This is a good idea. Right now, it fails after the terraform infrastructure has already been created. We can add the check much earlier.

Reply:
Added a docker image inspect command to check if the image already exists.